tests/integ/modin/groupby/test_groupby_apply.py

```diff
@@ -877,7 +874,7 @@ def test_group_apply_return_df_from_lambda(self):
             pd.DataFrame(diamonds_pd),
             diamonds_pd,
             lambda diamonds: diamonds.groupby("cut").apply(
-                lambda x: x.sort_values("price", ascending=False).head(5),
+                lambda x: x.nlargest(5, columns=["price"]),
```

Comment:
this does not fix the issue. pandas maintain the order from the original frame but we didn't.

Reply:
the `sort_values` is running inside pandas. pandas `sort_values` isn't guaranteed to be `stable` with the default `kind`. In the current version of the code, neither Snowpark pandas nor pandas is expected to necessarily maintain order.

Reply:
can we just set `kind = 'mergesort'`?

Reply:
Thank for helping this BTW

Reply:
done.